lewis/adapters/epics.py

```diff
@@ -101,6 +125,10 @@ def process_pv_updates(self, force=False):
             if self._timers[pv] >= pv_object.poll_interval or force:
                 try:
                     self.setParam(pv, getattr(self._target, pv_object.property))
+
+                    if pv_object.meta_data_property is not None:
+                        self.setParamInfo(pv, getattr(self._target, pv_object.meta_data_property))
```

Comment:
Thanks, I was not aware of this! I'm not sure it's an issue either, I agree on keeping an eye on it and seeing if it becomes a problem at some point.